src/mito2/src/engine.rs

```diff
@@ -111,8 +111,7 @@ impl MitoEngine {
         self.inner.handle_query(region_id, request)
     }
 
-    #[cfg(test)]
-    pub(crate) fn get_region(&self, id: RegionId) -> Option<crate::region::MitoRegionRef> {
+    pub fn get_region(&self, id: RegionId) -> Option<MitoRegionRef> {
```

Comment:
This method is unexpected to be public. We should not expose the `MitoRegion` except in tests.